discretize/_extensions/tree_ext.pyx

```diff
@@ -521,9 +521,9 @@ cdef class _TreeMesh:
         ----------
```

Comment:
In case you are not seeing on your side - I am getting a lot of warnings when compiling the code
![image](https://user-images.githubusercontent.com/55204635/204149452-6023c719-247e-4d5a-9a9f-5c8d2af4dff7.png)